src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -177,8 +221,18 @@ public void Save(ModelSaveContext ctx)
             ctx.SetVersionInfo(GetVersionInfo());
 
             ctx.SaveBinaryStream("OnnxModel", w => { w.WriteByteArray(Model.ToByteArray()); });
-            ctx.SaveNonEmptyString(_args.InputColumn);
-            ctx.SaveNonEmptyString(_args.OutputColumn);
+            //ctx.SaveNonEmptyString(_args.InputColumn);
+            //ctx.SaveNonEmptyString(_args.OutputColumn);
+
+            _host.AssertNonEmpty(Inputs);
```

Comment:
>AssertNonEmpty [](start = 18, length = 14)

CheckNonEmpty

Reply:
done

---
In reply to: [233267218](https://github.com/dotnet/machinelearning/pull/1586#discussion_r233267218) [](ancestors = 233267218)